generator/fluentReportsGenerator.js

```diff
@@ -5146,7 +5224,20 @@ class frBandElement extends frPrint { // jshint ignore:line
     }
 
     set border(val) {
-        this._border = val;
+        if(typeof val === "string" || typeof val === "number"){
```

Comment:
Is this from a Different PR; did you get something merged improperly.

Reply:
Pretty this was from a diff PR